Artifacts/linux-yum-package/Artifactfile.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+    "title": "Yum",
+    "description": "Install packages on a Linux Azure DevTest Lab VM via YUM",
+    "tags": [
+        "Package Management",
+        "Yum",
+        "install",
+        "Linux"
+    ],
+    "iconUri": "http://fedoraproject.org/w/uploads/7/77/Echo-package-48px.png",
+    "targetOsType": "Linux",
+    "parameters": {
+        "packages": {
+            "type": "string",
+            "displayName": "Packages",
+            "allowEmpty": false,
+            "description": "The packages to install on the VM, separated by spaces. All of the package declarations supported by yum are supported here as well, see http://man7.org/linux/man-pages/man8/yum.8.html for an in-depth look at the package specifications available."
+        },
+        "update": {
+            "type": "string",
+            "displayName": "Run Update First",
+            "defaultValue": "true",
+            "description": "Run yum update prior to running the full install command. Allowed values are 'true' and 'false', or blank (which will evaluate to false). Note that no additional options are passed to the update command."
```

Comment:
You can use the allowedValues json property to specify only true/false are allowed